### PR TITLE
Met and leps

### PR DESCRIPTION
**Pull request by lucalavezzo**
- fixed MET filters, thanks @Jennyggg for the spot!
- added lepton counters
- couple fixes to make code compatible with the naming scheme of the private samples
- some minor plotting adjustments

### CODE REVIEW

File: workflows/SUEP_coffea.py [hunk 1]
```diff
@@ -224,8 +224,56 @@ def getScoutingTracks(self, events):
         tracks = ak.packed(Cleaned_cands)
         return tracks, Cleaned_cands
 
+    def getLooseLeptons(self, events):
+        
+        muons = ak.zip(
```
Comment (chadfreer) [inline workflows/SUEP_coffea.py:229]: this collections is what I mean.

File: workflows/SUEP_coffea.py [hunk 2]
```diff
@@ -224,8 +224,56 @@ def getScoutingTracks(self, events):
...
+                "pt": events.Muon.pt,
+                "eta": events.Muon.eta,
+                "phi": events.Muon.phi,
+                "mass": events.Muon.mass,
+                "charge": events.Muon.pdgId / (-13),
+            },
+            with_name="Momentum4D",
+        )
+
+        electrons = ak.zip(
```
Comment (chadfreer) [inline workflows/SUEP_coffea.py:240]: and here

File: workflows/SUEP_coffea.py [hunk 3]
```diff
@@ -363,7 +414,7 @@ def analysis(self, events, do_syst=False, col_label=""):
         # golden jsons for offline data
         if not self.isMC and self.scouting != 1:
             events = applyGoldenJSON(self, events)
-        events, electrons, muons = ZH_utils.selectByLeptons(self, events, lepveto=True)
+        events, _, _ = ZH_utils.selectByLeptons(self, events, lepveto=True)
```
Comment (chadfreer) [inline workflows/SUEP_coffea.py:417]: And return this

File: workflows/SUEP_coffea.py (hunk 2 above)
Comment (lucalavezzo) [reply workflows/SUEP_coffea.py]: last push fixes this

File: workflows/SUEP_coffea.py (hunk 1 above)
Comment (lucalavezzo) [reply workflows/SUEP_coffea.py]: last push fixes this

File: workflows/SUEP_coffea.py [hunk 4]
```diff
@@ -224,8 +224,56 @@ def getScoutingTracks(self, events):
         tracks = ak.packed(Cleaned_cands)
         return tracks, Cleaned_cands
 
+    def getLooseLeptons(self, events):
+        
+        mlooseMuons = ak.zip(
```
Comment (chadfreer) [inline workflows/SUEP_coffea.py:229]: remove m

**Pull request merged**